packages/ui/src/components/va-hover/VaHover.vue

```diff
@@ -1,42 +1,44 @@
 <template>
-  <div @mouseenter="onMouseEnter" @mouseleave="onMouseLeave">
```

Comment:
I already refactored this component. Sorry, I thought I marked it as refactored here: https://github.com/epicmaxco/vuestic-ui/issues/1397#issuecomment-1016427099

This was required for VaCarousel https://github.com/epicmaxco/vuestic-ui/pull/1540/files#diff-b0e9176ecc1c0040b537f9886d4f3c32660bfd15d62135d43ef57902842a9693

Can I ask you to rework your changes in VaHover component to prevent conflicts?

Reply:
Let's resolve this conflict later.